Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/PolDiffILLReduction.py

```diff
@@ -0,0 +1,744 @@
+                 or (self._method_data_structure == '10p' and entry_no % 10 == 0) ):
+                correction_ws = self._match_attenuation_workspace(entry.name(), attenuation_ws)
+            ApplyPaalmanPingsCorrection(SampleWorkspace=entry,
+                                        CorrectionsWorkspace=correction_ws,
+                                        CanWorkspace=mtd[container_ws][entry_no],
+                                        OutputWorkspace=entry)
+        if self.getPropertyValue('ClearCache'):
+            DeleteWorkspaces(WorkspaceList=[correction_ws, attenuation_ws])
+        return sample_ws
+
+    def _data_structure_helper(self):
+        nMeasurements = 0
+        if self._method_data_structure == '10p':
+            nMeasurements = 10
+        elif self._method_data_structure == 'XYZ':
+            nMeasurements = 6
+        elif self._method_data_structure == 'Uniaxial':
+            nMeasurements = 2
+
+        return nMeasurements
+
+    def _normalise_vanadium(self, ws):
+        """Performs normalisation of the vanadium data to the expected cross-section."""
+
+        vanadium_expected_cross_section = 0.404 # barns
+        CreateSingleValuedWorkspace(DataValue=3.0 * vanadium_expected_cross_section
+                                    * self._sampleAndEnvironmentProperties['NMoles'].value,
+                                    OutputWorkspace='norm')
+        if self.getPropertyValue('OutputTreatment') == 'Sum':
+            tmp_name = '{}_1'.format(self.getPropertyValue('OutputWorkspace'))
+            RenameWorkspace(InputWorkspace=mtd[ws][0].name(), OutputWorkspace=tmp_name)
+            to_remove = ['norm']
+            for entry_no in range(1, mtd[ws].getNumberOfEntries()):
+                ws_name = mtd[ws][entry_no].name()
+                Plus(LHSWorkspace=tmp_name, RHSWorkspace=ws_name, OutputWorkspace=tmp_name)
+                to_remove.append(ws_name)
+            # expected total cross-section of unpolarised neutrons in V is 1/3 * sum of all measured c-s,
+            # and normalised to 0.404 barns times the number of moles of V:
+            Divide(LHSWorkspace='norm', RHSWorkspace=tmp_name, OutputWorkspace=tmp_name)
+            DeleteWorkspaces(WorkspaceList=to_remove)
+            GroupWorkspaces(InputWorkspaces=tmp_name, OutputWorkspace=ws)
+        else:
+            if self.getPropertyValue('OutputTreatment') == 'Average':
+                self._merge_polarisations(ws, average_detectors=True)
+            Divide(LHSWorkspace='norm', RHSWorkspace=ws, OutputWorkspace=ws)
+            DeleteWorkspace(Workspace='norm')
+        return ws
+
+    def _set_units(self, ws, process):
+        if process == 'Sample' and self.getPropertyValue('OutputTreatment') in  ['Average','Sum']:
+            self._merge_polarisations(ws, average_detectors=(self.getPropertyValue('OutputTreatment') == 'Average'))
+
+        for entry in mtd[ws]:
+            unit = 'dSigma / dOmega (detector number)'
```

Comment:
To be consistent with the rest, this should be in latin `d\sigma / d\omega` and I am not sure what is in parentheses.
There it should be eventually the unit, e.g. `(barn/steradian/formula unit)` or something like that.